packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -679,6 +661,33 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		return table;
 	}
 
+	public summarize(telemetryContext?: ITelemetryContext): ISummaryTreeWithStats {
```

Comment:
This is just a move of the summarize function here. There are no changes to the function.